developers/weaviate/tutorials/connect.mdx

```diff
@@ -9,92 +9,95 @@ import WCSWithoutAuthentication from '/_includes/code/wcs.without.authentication
 
-### For WCS users
+### Where to find connection details
 
-You can find the cluster URL, authentication status and enabled modules for each instance, in the Weaviate Cloud Services dashboard.
+**Weaviate Cloud Services (WCS)** If you are using Weaviate Cloud Services (WCS), the dashboard has a "Details" button. Click the button to get the following information:
 
-### For Docker/Kubernetes deployments
+ - The cluster URL
+ - Authentication type
+ - API keys, if enabled
+ - Module details
 
-You can find the relevant configuration in the configuration files, such as `docker-compose.yml` for Docker Compose, or `values.yaml` in the Helm chart.
+**Docker and Kubernetes** If you are using Docker or Kubernetes, your container configuration files have the connection details. 
 
-For Docker instances, your URL will likely be `http://localhost:8080` unless specified otherwise.
+- For Docker Compose, see `docker-compose.yml`.
+- For Kubernetes, see `values.yaml` in the Helm chart.
 
-:::tip For Embedded Weaviate
-If you are using Embedded Weaviate, you do not need to specify a URL at the client instantiation step, and most likely will not need any authentication.
-<br/>
-<p>
-So, you can simply instantiate with the following.
-</p>
+Docker instances often use the `http://localhost:8080` address.
 
-<EmbeddedInstantiationSimple/>
+**Embedded Weaviate** If you are using Embedded Weaviate, you do not need to specify a URL when you create the client. Embedded instances usually don't need authentication details. For details, see the [Embedded Weaviate documentation](../installation/embedded.md).
 
-For the default configuration, and more about the option in general, see the [Embedded Weaviate documentation](../installation/embedded.md).
-:::
+## Connecting to Weaviate
 
-# Connecting to Weaviate
+### Without authentication
 
-## Without Authentication
-
-:::note Not recommended
-This set up is not recommended in general as anyone can access your Weaviate instance as long as they have the URL.
+:::warning Use authentication
+Do not expose your Weaviate instance to the public internet if authentication is not enabled. If you do not enable authentication, anyone who has the URL can access your Weaviate instance.
 :::
 
+To connect to Weaviate with a client application, follow these examples: 
+
 <WCSWithoutAuthentication/>
 
-## With authentication enabled
+### With authentication
 
-If you do have authentication enabled, you can log in with an API key (recommended), or with OIDC.
+If authentication is enabled, use a Weaviate API key or OIDC to connect. For details, see [Authentication](../configuration/authentication).
 
-### With API key authentication (recommended)
+#### Weaviate API keys
 
-:::info This is your Weaviate API key
-This is **not** the place to specify the API key for services such as Cohere, Hugging Face or OpenAI.
-:::
+To connect to Weaviate with a client application that uses a Weaviate API key, follow these examples.
```

Comment:
To be a little pedantic, I would prefer:

`To authenticate against an Weaviate instance with an API key, follow these examples.`